systemtest/driver/utils.py

```diff
@@ -91,3 +92,9 @@ def copy_files(*, sources: List[pathlib.Path], dest: pathlib.Path):
     for path in sources:
         if path.is_file():
             shutil.copy2(path, dest)
+
+def pairwise(iterable):
+    "s -> (s0,s1), (s1,s2), (s2, s3), ..."
```

Comment:
This should be:
```
"""
s -> (s0,s1), (s1,s2), (s2, s3), ...
"""
```
While this is technically a doc string, i.e. a string literal as the first code inside the function, PEP-257 strongly recommends to use triple quoted strings for doc strings

Reply:
done